crates/core/src/yarn_fn/function_registry.rs

```diff
@@ -115,11 +124,44 @@ mod tests {
 
         functions.register_function("test", |a: f32| a);
         let function = functions.get("test").unwrap();
-        let result: f32 = function.call(to_function_params([1.0])).try_into().unwrap();
+        let result: f32 = function
+            .call(
+                to_function_params([1.0]),
+                #[cfg(feature = "bevy")]
+                &mut World::default(),
+            )
+            .try_into()
+            .unwrap();
 
         assert_eq!(result, 1.0);
     }
 
+    #[cfg(feature = "bevy")]
+    #[test]
+    fn can_access_bevy_world() {
```

Comment:
Could you also add tests for 
- Using only `In` params and
- Using a mix of `In` and `Query`

?